Configuration/AlCa/python/autoCond.py

```diff
@@ -68,21 +68,21 @@
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
     'phase1_2018_cosmics_peak'     : '123X_upgrade2018cosmics_realistic_peak_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Phase1 2022
-    'phase1_2022_design'           : '125X_mcRun3_2022_design_v2',
+    'phase1_2022_design'           : '125X_mcRun3_2022_design_v4',
     # GlobalTag for MC production with realistic conditions for Phase1 2022
-    'phase1_2022_realistic'        : '125X_mcRun3_2022_realistic_v4',
+    'phase1_2022_realistic'        : '125X_mcRun3_2022_realistic_v5',
     # GlobalTag for MC production (cosmics) with realistic conditions for Phase1 2022,  Strip tracker in DECO mode
-    'phase1_2022_cosmics'          : '125X_mcRun3_2022cosmics_realistic_deco_v4',
+    'phase1_2022_cosmics'          : '125X_mcRun3_2022cosmics_realistic_deco_v5',
     # GlobalTag for MC production (cosmics) with perfectly aligned and calibrated detector for Phase1 2022, Strip tracker in DECO mode
-    'phase1_2022_cosmics_design'   : '125X_mcRun3_2022cosmics_design_deco_v2',
+    'phase1_2022_cosmics_design'   : '125X_mcRun3_2022cosmics_design_deco_v3',
     # GlobalTag for MC production with realistic conditions for Phase1 2022 detector for Heavy Ion
     'phase1_2022_realistic_hi'     : '125X_mcRun3_2022_realistic_HI_v6',
```

Comment:
this one also looks a candidate for clean-up, but do see the discussion going on at the data PR: https://github.com/cms-sw/cmssw/pull/40011#issuecomment-1306949965

Reply:
done